server/server.js

```diff
@@ -148,6 +148,15 @@ module.exports = function(port, tweetSearcher, googleAuthoriser) {
         }
     });
 
+    app.post("/admin/tweets/approvedTweetsOnlyStatus", function(req, res) {
+        try {
```

Comment:
I think our error handling in general could do with an overhaul so I'm happy to just leave it here.